lib/Runtime/ByteCode/FuncInfo.cpp

```diff
@@ -427,11 +427,13 @@ void FuncInfo::OnEndVisitFunction(ParseNode *pnodeFnc)
     this->SetCurrentChildFunction(nullptr);
 }
 
-void FuncInfo::OnStartVisitScope(Scope *scope)
+bool FuncInfo::OnStartVisitScope(Scope *scope)
```

Comment:
I would rather prefer *out* variable from the function, instead of returning bool from the function. like (Scope *scope, bool *isMergedScope).